src/aura/STG_CMP_Base/STG_CMP_BaseHelper.js

```diff
@@ -81,6 +56,14 @@
 		}
 	},
 	
+	addPrefix : function(namespacePrefix, key) {
```

Comment:
Actually, we are manipulating custom settings fields, and the namespace needs to have the double underscore. This is done in https://github.com/SalesforceFoundation/HEDAP/blob/master/src/pages/STG_Settings.page#L29.